corehq/apps/events/views.py

```diff
@@ -448,13 +447,13 @@ def poll_mobile_worker_attendee_progress(request, domain, download_id):
         return HttpResponseServerError()
 
     if mobile_worker_attendees_enabled(domain):
-        context['success_message'] = _("Disabling mobile workers complete!")
```

Comment:
Instead of using separate if/else blocks for enabling and disabling, you could create a variable:
```
status_text = "Disabling"
if mobile_worker_attendees_enabled(self.domain):
    status_text = "Enabling"
```
You can then simply have the messages as `context['success_message'] = _(f"{status_text} mobile workers complete!")` for example.